src/reducers/entities/channels.ts

```diff
@@ -310,7 +310,7 @@ function myMembers(state = {}, action) {
     }
 }
 
-function membersInChannel(state = {}, action) {
+export function membersInChannel(state = {}, action) {
```

Comment:
Exporting functions that are not being publicly used and is only being exported for testing is considered an anti-pattern. 
https://medium.com/@macsikora/how-to-test-private-functions-of-es6-module-fb8c1345b25f

Perhaps you could import the reducer in the test and invoke that?

kinda like 
```
import reducer from 'reducers/entities/channels';
...
reducer({}, action);

```

Reply:
Yes, I agree, now is done. I only changed my implementation because this is a PR that is going to be cherry-picked so I prefer to not generate too much complexity.

Reply:
I think testing using the `reducer` would've been less work because you don't need to export the function, but I can understand that it might increase risk or unknowns, because it's not being done that way currently.